src/containers/pages/PractitionerViews/CreateEditPractitioner/tests/index.test.tsx

```diff
@@ -0,0 +1,264 @@
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import { mount, shallow } from 'enzyme';
+import { createBrowserHistory } from 'history';
+import React from 'react';
+import Helmet from 'react-helmet';
+import { Provider } from 'react-redux';
+import { Router } from 'react-router';
+import ConnectedCreateEditPractitionerView, { CreateEditPractitionerView } from '..';
+import {
+  EDIT,
+  EDIT_PRACTITIONER_URL,
+  NEW,
+  OPENSRP_PRACTITIONER_ENDPOINT,
+  PRACTITIONER,
+} from '../../../../../constants';
+import store from '../../../../../store';
+import * as practitionerDucks from '../../../../../store/ducks/opensrp/practitioners';
+import * as fixtures from '../../../../../store/ducks/tests/fixtures';
+
+reducerRegistry.register(practitionerDucks.reducerName, practitionerDucks.default);
+
+// tslint:disable-next-line: no-var-requires
+const fetch = require('jest-fetch-mock');
+
+const history = createBrowserHistory();
+
+describe('src/containers/pages/CreateEditOrganization', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    store.dispatch(practitionerDucks.removePractitionerRolesAction);
+    store.dispatch(practitionerDucks.removePractitionersAction);
+  });
+
+  it('renders EditTeamView without crashing', () => {
```

Comment:
Is a practitioner the same thing as a team?

Reply:
nope

Reply:
@p-netm then perhaps we should not reference teams in this file?